ckanext/data_qld/user_creation/validators.py

```diff
@@ -6,8 +6,10 @@
 def data_qld_user_name_validator(key, data, errors, context):
     user = toolkit.c.userobj
     is_sysadmin = user is not None and user.sysadmin
-
-    if not is_sysadmin and 'publisher' in data[key].lower():
+    # If the username is existing and haven't changed or its from reset password form, skip validation
+    if user is None or user.name.lower() == data[key].lower():
```

Comment:
Can you also do the same for display name skip so that user profiles can be edited when username or displayname/full name is not altered and is set to publisher etc